server/test/controllers/admin/AdminProgramImageControllerTest.java

```diff
@@ -61,7 +62,7 @@ public void index_ok_get() throws ProgramNotFoundException {
 
     Result result =
         controller.index(
-            addCSRFToken(fakeRequest().method("GET")).build(),
+            addCSRFToken(fakeRequestBuilder().method("GET")).build(),
```

Comment:
**question (non-blocking):** Same question here.  Aren't we adding the CSRFToken twice?

Reply:
Same answer!